src/main/java/seedu/address/model/person/Address.java

```diff
@@ -21,6 +21,15 @@
 
     public final String value;
 
+    //@@author alexfoodw
+    /**
+     * Constructs a blank address field
+     */
+    public Address() {
+        this.value = " ";
```

Comment:
Might want to put blank address as a constant

Reply:
Alternatively, we can just let the Person take the address as a null value and handle based on that